isdown/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "98d8eda3-7bc5-4067-8ff6-63d11d0132e2",
+  "app_id": "isdown",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "IsDown helps companies monitor all third-party status pages in one place",
+    "title": "IsDown",
+    "media": [
+      {
+        "media_type": "image",
```

Comment:
This image will get cut off in our media carousel - is there a way to make the image have a 16:9 aspect ratio so it does not get cut off?

Reply:
Thank you @anawishnoff and @emarsha94. I've changed the image. Let me know if I got it wrong, please.

Reply:
Hi @ntomas the image requirements are: 
- Width: 1440px
- Minimum height: 810px
- Maximum height: 2560px

This image still exceeds the requirements. Can you update it to meet the above? Thanks!

Reply:
Hi, @emarsha94. Hopefully I got it right this time. Thank you!